lib/host-sp-messages/src/lib.rs

```diff
@@ -41,7 +51,11 @@ pub enum HostToSp {
     // Microoptimization: insert a dummy variant first, so we never serialize a
     // command value of `0` to make COBS's life slightly easier.
     _Unused,
+    RequestReboot,
```

Comment:
Should we either (a) explicitly set values here or (b) use a shared crate with the host side?

Reply:
The host side is being written in C, which rules out (b). :-D

Explicitly listing values that can be cross referenced with RFD 316 / the C header is a good idea though, I'll do that.

Reply:
Whoops, can't actually do that because some of these variants have associated data. I added a block comment and unit tests for both `HostToSp` and `SpToHost` in ebe90e9a instead.